flow-path/src/main/java/flow/path/PathContainer.java

```diff
@@ -64,31 +63,52 @@ public void restoreViewState(View view) {
 
   private final int tagKey;
 
+  /**
+   * @param tagKey an id used to store bookkeeping info on container views via {@link
+   * View#setTag(int, Object)}
+   */
   protected PathContainer(int tagKey) {
     this.tagKey = tagKey;
   }
 
-  public final void executeTraversal(PathContainerView view, Flow.Traversal traversal,
+  /**
+   * @deprecated use {@link #executeFlowTraversal(ViewGroup, Flow.Traversal,
+   * Flow.TraversalCallback)}
+   */
+  @Deprecated public final void executeTraversal(PathContainerView view, Flow.Traversal traversal,
       final Flow.TraversalCallback callback) {
+    final View oldChild = view.getCurrentChild();
+    ViewGroup containerView = view.getContainerView();
     ViewState viewState = traversal.destination.currentViewState();
-    doShowPath(view, traversal.destination.<Path>top(), traversal.direction, viewState, callback);
+    doShowPath(traversal.destination.<Path>top(), containerView, oldChild, traversal.direction,
+        viewState, callback);
+  }
+
+  /**
+   * Replaces the contents of a given {@link ViewGroup} with a new view inflated from
+   * a {@link Flow.Traversal}.
+   */
+  public final void executeFlowTraversal(ViewGroup container, Flow.Traversal traversal,
```

Comment:
Name change rather than override s.t. I don't break existing code. A straight override of `executeTraversal` is ambiguous for any `ViewGroup` that implements `PathContainerView`.